neurobooth_os/iout/metadator.py

```diff
@@ -196,30 +196,41 @@ def _get_instruction_kwargs(instruction_id, conn):
     return dict_instr
 
 
+def log_task_params(conn, log_task_id, task_param_dictionary):
```

Comment:
Add type annotations where known? E.g., we at least have `Dict[str, Any]` for the param dict and one of the returns.